app/models/intervention.rb

```diff
@@ -11,6 +11,16 @@ class Intervention < StudyRelationship
         { name: :intervention_type, value: :type },
         { name: :name, value: :name },
         { name: :description, value: :description }
+      ],
+      children: [
+        {
+          table: :intervention_other_names,
+          root: [:otherNames],
+          columns: [
+            { name: :intervention_id, value: nil },
```

Comment:
remove this column specification since this is implied from the child relationship

Reply:
done